deploy/catalog/templates/apiserver.yaml

```diff
@@ -18,15 +18,15 @@ spec:
         - {{ default "etcd" .Values.storageType }}
         - -v
         - {{ default "\"10\"" .Values.verbosity }}
-        {{ if eq .Values.storageType "etcd" }}- --etcd-servers {{ end }}
-        {{ if eq .Values.storageType "etcd" }}- http://localhost:2379 {{ end }}
+        {{ if or (empty .Values.storageType) (eq .Values.storageType "etcd") }}- --etcd-servers {{ end }}
+        {{ if or (empty .Values.storageType) (eq .Values.storageType "etcd") }}- http://localhost:2379 {{ end }}
```

Comment:
@agonzalezro how about using `default` to reduce logic.

```
{{ if eq (default .Values.storageType "etcd") "etcd" -}}
- --etcd-servers
- http://localhost:2379
{{- end }}
```

Same with the below